### ISSUE TITLE
[Feature Request] Custom color formatting in info lines

### ISSUE THREAD

**Issue by Jakub-Wilk** [maruohon/minihud#198]
I haven't seen an issue for that yet, so just adding in case you haven't seen that yet

I think having the option to format the lines with the usual Minecraft color codes:

![image](https://user-images.githubusercontent.com/45641296/149202441-8a156be2-208a-4f54-83fc-deacf8dbf731.png)

would be really useful, and could really improve readability. Implementing the codes on the bottom would be a nice bonus (except &r of course, that's important), but that seems quite a bit harder.

Here's a little (really ugly) mockup I did:

![2022-01-12_19 16 10](https://user-images.githubusercontent.com/45641296/149199482-abb42ee2-e231-4ca7-ba54-a92fc4b4450d.png)

**Comment by maruohon** [comment]
Issues #70, #88 and #144 are quite directly related.

So yes this is planned, since the fully customizable info lines will support format codes and any number of the info substitution strings to build your own info lines.

**Comment by alexwrinner** [comment]
A cool mod thanks [👍](https://trustanalytica.com/online/top-cheap-essay-writing-services)

**Comment by alexwrinner** [comment]
I think its cool                                                                [![111](https://github.com/maruohon/minihud/assets/143604329/e9acfa45-da75-4a62-b640-40d23234b856)](https://myessaytyper.ai/) [![111](https://github.com/maruohon/minihud/assets/143604329/e9acfa45-da75-4a62-b640-40d23234b856)](https://www.linkedin.com/pulse/5-best-online-casinos-australia-casino-reviews)

**Comment by maruohon** [comment]
@alexwrinner Why did your earlier message have a thumbs up emoji with an ad link? (a link to some weird-ish web site)

**Comment by alexwrinner** [comment]
spam 80lv

**Comment by pointydev** [comment]
Is this still planned? I'd love to be able to colour my block position following the colour scheme of the debug crosshair.

**Comment by maruohon** [comment]
Yes it's still planned, but my work on the mod rewrites has been almost non-existent in the past couple of years...
